dashboard/pages/dependency_graph.tsx

```diff
@@ -27,45 +27,53 @@ import Title from "../components/Title"
 import useFetch from "../lib/api/fetch"
 import {
   Relation,
+  getRelationDependencies,
   getRelations,
   relationIsStreamingJob,
 } from "../lib/api/streaming"
 import { RelationPoint } from "../lib/layout"
 
 const SIDEBAR_WIDTH = "200px"
 
-function buildDependencyAsEdges(list: Relation[]): RelationPoint[] {
+function buildDependencyAsEdges(
+  list: Relation[],
+  relation_deps: Map<number, number[]>
+): RelationPoint[] {
   const edges = []
   const relationSet = new Set(list.map((r) => r.id))
   for (const r of reverse(sortBy(list, "id"))) {
     edges.push({
       id: r.id.toString(),
       name: r.name,
       parentIds: relationIsStreamingJob(r)
-        ? r.dependentRelations
-            .filter((r) => relationSet.has(r))
-            .map((r) => r.toString())
+        ? relation_deps.has(r.id)
+          ? relation_deps
+              .get(r.id)
+              ?.filter((r) => relationSet.has(r))
+              .map((r) => r.toString())
+          : []
         : [],
       order: r.id,
       width: nodeRadius * 2,
       height: nodeRadius * 2,
       relation: r,
     })
   }
-  return edges
+  return edges as RelationPoint[]
 }
 
 export default function StreamingGraph() {
   const { response: relationList } = useFetch(getRelations)
+  const { response: relationDeps } = useFetch(getRelationDependencies)
```

Comment:
Could we add comments indicating that `dependentRelations` is deprecated and should not be used?